src/Chat/Client.js

```diff
@@ -73,11 +73,11 @@ function handleOpen(options) {
   this.send(`CAP REQ :${constants.CAPABILITIES.join(' ')}`, { priority })
 
   // Authenticate.
-  this.send(`PASS ${options.oauth}`, { priority })
+  this.send(`PASS ${options.password}`, { priority })
```

Comment:
My fear with changing this from `oauth` to `password` is that users will try their actual _Twitch password_ instead of the required OAuth token. If you think `oauth` is unclear, would `token` be better?